crates/light-client-prover/src/circuit/mod.rs

```diff
@@ -19,6 +19,12 @@ use sov_rollup_interface::Network;
 
 use crate::circuit::method_id_verifier::verify_method_id_security_council;
 
+/// Size of a compressed public key in bytes.
+pub const SECURITY_COUNCIL_COMPRESSED_PUBKEY_SIZE: usize = 33;
+
+/// Total number of security council members.
+pub const SECURITY_COUNCIL_MEMBERS: usize = 5;
```

Comment:
```suggestion
pub const SECURITY_COUNCIL_N_MEMBERS: usize = 5;
```
Or something better but it wasn't clear

Reply:
SECURITY_COUNCIL_MEMBER_COUNT